setup.cfg

```diff
@@ -1,2 +1,2 @@
 [flake8]
-max-line-length = 99
+max-line-length = 105
```

Comment:
This link explains why should be have max length limit: https://www.python.org/dev/peps/pep-0008/#maximum-line-length.

We should not update this any number as per requirement. By setting setup.cfg, we are disabling the limit for entire file. Along with import, I see rest of the code also not following max length limit. 
To avoid refactoring for rest of the code later, I would suggest to suppress the import flake8 warning. https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html

Reply:
Thanks for the suggestion - will update accordingly!

Reply:
Make sure that you remove setup.cfg before merging this PR.